activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -325,6 +325,26 @@ def load_schema(configuration, format = ActiveRecord::Base.schema_format, file =
         Migration.verbose = verbose_was
       end
 
+      def dump_schema(configuration, format = ActiveRecord::Base.schema_format, spec_name = "primary")
```

Comment:
@eileencodes Should we `nodoc` this method in similar way as https://github.com/rails/rails/pull/36416/files#diff-1452ba37bae4c4676b4288be7db09b8dL307 is? I missed this detail.

Reply:
Yes, please do.